app/components/articleSearch/components/searchItem/infoList.tsx

```diff
@@ -56,9 +56,9 @@ const InfoList = (props: IInfoListProps) => {
       </div>
       <span className={styles.explanation}>Cited Paper Avg IF</span>
       <span className={styles.citedPaperAvgIF}>{props.citedPaperAvgIF}</span>
-      <div className={styles.separatorLine} />
+      {/* <div className={styles.separatorLine} />
```

Comment:
also.

Reply:
It will be used later(Maybe when pluto point is useful).
So I want to remain this.